data/campaigns/World_Conquest/lua/game_mechanics/unittypedata.lua

```diff
@@ -350,51 +417,45 @@ type_infos = {
 		} }
 	},
 	["Woodsman"] = { 
+		-- po: a Woodsman joins a player's side
 		founddialogue=_"Ho there, friends! I am but a soldier of humble circumstances, yet long have I dreamed of joining great wars beyond our shores. Let me join your mission!",
 		name="Woodsman",
 	},
 	["Dune Herbalist"] = { 
+		-- po: a Dune Herbalist joins a player's side
 		founddialogue=_"Long have I studied the ways of lore, and I have much wisdom to offer. Allow me to guide you on your quest and together we will accomplish great things!",
 		image="units/dunefolk/herbalist.png",
 		name="Dune Herbalist",
 	},
 	["Dune Soldier"] = { 
+		-- po: a Dune Soldier joins a player's side
 		founddialogue=_"I too have come a long way to this strange land. Perhaps we were destined to join blades here.",
 		image="units/dunefolk/soldier.png",
 		name="Dune Soldier",
 	},
 	["Dune Rover"] = { 
+		-- po: a Dune Rover joins a player's side
```

Comment:
Same as Spearman and Woodsman.

Reply:
While it doesn't quite look right, I think this would be as helpful as it can be for translators, and less likely to break if there are revisions to the lines.

All the hints for a line accumulate, so in Poedit's hint box it will say
> a Spearman joins a player's side
> a Woodsman joins a player's side
> a Dune Rover joins a player's side

Reply:
I assumed you wanted to combine the hints since you did it for
> po: speaker is a Gryphon, just after a naga or merfolk joins their side

for example. I don't mind either way.